### PR TITLE
GFT

### PR DESCRIPTION
**Pull request by AleCalcagno**
Modify frontend

### CODE REVIEW

File: aegis-web/yo/app/index.html [hunk 1]
```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" href="bower_components/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css" />
     <link rel="stylesheet" href="bower_components/bootstrap-switch/dist/css/bootstrap3/bootstrap-switch.css" />
     <link rel="stylesheet" href="bower_components/angular-ui-select/dist/select.css" />
-    <link rel="stylesheet" href="bower_components/angular-bootstrap/ui-bootstrap-csp.css" />
```
Comment (cbotsikas) [inline aegis-web/yo/app/index.html:60]: Why is this removed? @gbikas16 You've added this, do we use it?

File: aegis-web/yo/app/index.html [hunk 2]
```diff
@@ -165,6 +163,8 @@
     <script src="bower_components/nvd3/build/nv.d3.js"></script>
     <script src="bower_components/angular-nvd3/dist/angular-nvd3.js"></script>
     <script src="bower_components/angular-bootstrap-toggle/dist/angular-bootstrap-toggle.min.js"></script>
+    <script src="bower_components/blob-polyfill/Blob.js"></script>
+    <script src="bower_components/file-saver.js/FileSaver.js"></script>
```
Comment (cbotsikas) [inline aegis-web/yo/app/index.html:167]: this is reverting https://github.com/aegisbigdata/hopsworks/commit/15d88cf0b416d50a74ffc396c01dacd85b7b1b37 by @michaelalemu. Why is that? The bower components exist so i guess there are other packages that require them.

File: aegis-web/yo/app/scripts/controllers/homeCtrl.js [hunk 3]
```diff
@@ -70,7 +70,8 @@ angular.module('hopsWorksApp')
                 {'name': 'Kafka', 'tip': 'Take a tour of Hopsworks by creating a project and running a Kafka job!'}
               ];
             };
-
+            
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/scripts/controllers/homeCtrl.js]: Unnecessary change.

File: aegis-web/yo/app/scripts/controllers/mainCtrl.js [hunk 4]
```diff
@@ -413,8 +413,10 @@ angular.module('hopsWorksApp')
             self.viewDetail = function (result) {
               if (result.localDataset) {
                 if (result.type === 'proj') {
+                    console.log(result.name)
                   ProjectService.getProjectInfo({projectName: result.name}).$promise.then(
                           function (response) {
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/scripts/controllers/mainCtrl.js:418]: Do not commit debug code.

File: aegis-web/yo/app/scripts/controllers/selectProjectCtrl.js [hunk 5]
```diff
@@ -91,6 +91,7 @@ angular.module('hopsWorksApp')
                 growl.error(error.data.errorMsg, {title: 'Error', ttl: 10000});
               });
             };
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/scripts/controllers/selectProjectCtrl.js:94]: Unnecessary change.

File: aegis-web/yo/app/scripts/services/ProjectService.js [hunk 6]
```diff
@@ -40,8 +40,11 @@
 'use strict';
 
 angular.module('hopsWorksApp')
+        
         .factory("ProjectService", ['$resource', function ($resource) {
+                
             return $resource(
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/scripts/services/ProjectService.js:47]: Unnecessary change.

File: aegis-web/yo/app/styles/search-result.css [hunk 7]
```diff
@@ -180,6 +180,7 @@
   left: 4px;
 }
 
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/styles/search-result.css:183]: Unnecessary file change.

File: aegis-web/yo/app/views/login.html [hunk 8]
```diff
@@ -37,16 +37,16 @@
   ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   -->
 
-<div class="alert alert-warning" ng-if="loginCtrl.announcement !== ''">
+<div class="alert alert-warning" ng-if="loginCtrl.announcement !== ''" style="margin-bottom: 100px;">
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/login.html:40]: This might be the wrong place to add the margin. If there is no announcement, the login form has zero space from the top. If your intention was to move the login form, apply style to the login form, not the alert.

File: aegis-web/yo/app/views/login.html [hunk 9]
```diff
@@ -37,16 +37,16 @@
...
 <div class="container content-wrapper" ng-if="!loginCtrl.secondFactorRequired" style="border-top:none">
   <div class="row">
     <div class="center-block" style="width: 320px;">
       <div class="panel">
         <div class="panel-heading text-center">
-          <a style="text-decoration:none; font-family: Lobster; font-size: 30px; margin-left: 5px; color: #fff;">Hopsworks Beta
-            <img alt="" class="block-center img-rounded" style="margin-left:10px" width="30" src="images/hops.png">
+          <a>
+            <img alt="" class="block-center img-rounded" style="margin-left:10px" width="100" src="images/aegis_logo.png">
           </a>
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/login.html:50]: Remove the hyperlink tag <a> altogether. Keep only the img inside the .panel-heading div.

File: aegis-web/yo/app/nav.html [hunk 10]
```diff
@@ -103,7 +103,9 @@
         <!--ng-show="showDatasets"-->
         <form class="navbar-form">
           <div class="col-xs-12 col-sm-4 col-md-4 input-group">
-            <input type="text" class="form-control dark" style="height: 44px; background-color: #6baec6; border: none; color: #fff"
+
+            <input type="text" class="form-control dark" style="height: 44px; background-color: #ffffff; border: none; border-top-left-radius: 10px; border-bottom-left-radius: 10px;"
+
```
Comment (cbotsikas) [inline aegis-web/yo/app/nav.html]: Unnecessary empty lines.

File: aegis-web/yo/app/nav.html [hunk 11]
```diff
@@ -103,7 +103,9 @@
         <!--ng-show="showDatasets"-->
         <form class="navbar-form">
           <div class="col-xs-12 col-sm-4 col-md-4 input-group">
-            <input type="text" class="form-control dark" style="height: 44px; background-color: #6baec6; border: none; color: #fff"
+
+            <input type="text" class="form-control dark" style="height: 44px; background-color: #ffffff; border: none; border-top-left-radius: 10px; border-bottom-left-radius: 10px;"
```
Comment (cbotsikas) [inline aegis-web/yo/app/nav.html:107]: Is this the desired result? 
image

File: aegis-web/yo/app/scripts/js/util.js [hunk 12]
```diff
@@ -194,3 +194,11 @@ function skipTrailingSlash(path) {
...
   e.stopPropagation();
 });
+
+jQuery(document).on('click', '.dropdown', function(e) {
+    if($(this).hasClass('open')) {
+        $(this).removeClass('open');
+    } else {
+        $(this).addClass('open');
+    }
+});
```
Comment (cbotsikas) [inline aegis-web/yo/app/scripts/js/util.js]: Why would you write this code when you could use Bootstrap's dropdown? 
Check your local style guide: http://localhost:9001/styleguide.html#dropdown

File: aegis-web/yo/app/views/search.html [hunk 13]
```diff
@@ -73,26 +73,37 @@
...
         <ul>
-          <li class="dropdown-header">Type</li>
-          <li><a class="filter-menu-uppercase"
+        <div class="dropdown">
+                <p>Type
+                    <button class="btn btn-default dropdown-toggle" type="button" id="dropdownType">
+                        
+                        <span class="caret"></span>
+                    </button>
+                </p>
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/search.html]: Have a look at your local styleguide.
Remove `<p>` keeping only the `<button>`. 
Put your text "Type" before `<span>` inside the `<button>`.
add  `data-toggle="dropdown"` to your `<button>`

File: aegis-web/yo/app/styles/form.css [hunk 14]
```diff
@@ -232,3 +232,7 @@ textarea.form-control {
 .dark::placeholder {
   color: #fff !important;
 }
+
+select {
+  text-transform: uppercase;
+}
```
Comment (cbotsikas) [inline aegis-web/yo/app/styles/form.css:234]: This applies to all `<select>` elements in aegis. Are you sure you want this?

**Review by cbotsikas — CHANGES_REQUESTED** [review]
See inline comments

File: aegis-web/yo/app/index.html (hunk 2 above)
Comment (michaelalemu) [reply aegis-web/yo/app/index.html:167]: I must admit that I’m not sure why these bower components had been removed/readded by me in first place...

File: aegis-web/yo/app/nav.html (hunk 11 above)
Comment (Ambradibi) [reply aegis-web/yo/app/nav.html:107]: Yes, but if you prefer another style tell me

File: aegis-web/yo/app/nav.html (hunk 10 above)
Comment (Ambradibi) [reply aegis-web/yo/app/nav.html]: removed space

File: aegis-web/yo/app/scripts/controllers/homeCtrl.js (hunk 3 above)
Comment (Ambradibi) [reply aegis-web/yo/app/scripts/controllers/homeCtrl.js]: removed space

File: aegis-web/yo/app/scripts/controllers/mainCtrl.js (hunk 4 above)
Comment (Ambradibi) [reply aegis-web/yo/app/scripts/controllers/mainCtrl.js:418]: Sorry I didn't notice. Removed

File: aegis-web/yo/app/scripts/controllers/selectProjectCtrl.js (hunk 5 above)
Comment (Ambradibi) [reply aegis-web/yo/app/scripts/controllers/selectProjectCtrl.js:94]: Removed space

File: aegis-web/yo/app/views/search.html (hunk 13 above)
Comment (Ambradibi) [reply aegis-web/yo/app/views/search.html]: ok, done

File: aegis-web/yo/app/scripts/js/util.js (hunk 12 above)
Comment (Ambradibi) [reply aegis-web/yo/app/scripts/js/util.js]: removed and corrected

File: aegis-web/yo/app/styles/form.css (hunk 14 above)
Comment (Ambradibi) [reply aegis-web/yo/app/styles/form.css:234]: removed

File: aegis-web/yo/app/index.html (hunk 1 above)
Comment (gbikas) [reply aegis-web/yo/app/index.html:60]: We need this for uib-tabset in walletTransactions.html

**Review by o-alex — APPROVED** [review]
Most of the changes relate to front end only (not rest endpoints/backend) so I will say the changes are fine with me.
Christos also addressed most of the styling/polluting issues like empty lines or spaces .

**Comment by cbotsikas** [comment]
I'm closing this PR in favor of PR https://github.com/aegisbigdata/hopsworks/pull/10
